xija/component/heat.py

```diff
@@ -869,7 +874,8 @@ def plot_solar_heat__pitch(self, fig, ax):
                 lines[i].set_data(self.P_pitches, P_vals[instr_name])
                 # lines[i * 2 + 1].set_data(self.P_pitches, P_vals[instr_name], '-b')
         else:
-            for instr_name, color in zip(self.instr_names, colors):
+            for i, instr_name in enumerate(self.instr_names):
+                color = colors[i]
```

Comment:
Why this change?

Reply:
Originally the number of colors was equal to the number of SIM-Z position ranges, which was 4. This allows it to be any number >= 2, but if we wanted it to be larger than 4 we'd need to make the `color` list longer.

Reply:
`zip` will stop when the smallest list finishes, so the original was fine. To be really correct here you could do:
```
zip(self.instr_names, itertools.cycle(colors))
```